### DISCUSSION TITLE
Chia v1.1.5 fails to install

### DISCUSSION THREAD

**Discussion by Nikolaperun** [Chia-Network/chia-blockchain#4341] [Support]
Hello,
I am trying to install chia blockchain (.exe) on a machine that will be used for farming only. System specs are:

Win10 Pro x86
CPU: Intel Pentium D (dual core) @2,8GHz
MoBo: Asus P5B
RAM: 3GB DDR2@667MHz
etc..

Are minimum system requirement met here, please?

Regards,
Nik

**Comment by Nikolaperun** [comment]
Just to add a detail, %temp% folder has no logs about this failed installation.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
